development/ts-migration-dashboard/scripts/build-app.ts

```diff
@@ -5,8 +5,10 @@ import { hideBin } from 'yargs/helpers';
 import chokidar from 'chokidar';
 import browserify from 'browserify';
 import pify from 'pify';
-import endOfStream from 'end-of-stream';
-import pump from 'pump';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error types/readable-stream.d.ts does not get picked up by ts-node
```

Comment:
That's weird. 
Also, while we're at it - why `import { pipeline } from 'readable-stream';` and not `import { pipeline } from 'node:streams';` ?

Reply:
In this particular case we could do it, but generally in the extension we should use `readable-stream` because `node:stream` would need to be polyfilled.

Reply:
`readable-stream` is specifically preferred for portability and being stable across runtimes and combinations of build-config/node-versions.

Reply:
> That's weird. Could it be a version mismatch between the package and the types?
https://github.com/MetaMask/metamask-extension/pull/21934#discussion_r1616311337